st3/lsp_utils/node_runtime.py

```diff
@@ -22,11 +22,11 @@
 
 IS_WINDOWS_7_OR_LOWER = sys.platform == 'win32' and sys.getwindowsversion()[:2] <= (6, 1)  # type: ignore
 
-NODE_RUNTIME_VERSION = '16.15.0'
+NODE_RUNTIME_VERSION = '18.18.1'
 NODE_DIST_URL = 'https://nodejs.org/dist/v{version}/{filename}'
 
-ELECTRON_RUNTIME_VERSION = '22.2.0'  # includes Node.js v16.17.1
-ELECTRON_NODE_VERSION = '16.17.1'
+ELECTRON_RUNTIME_VERSION = '26.4.0'  # includes Node.js v18.16.1
+ELECTRON_NODE_VERSION = '18.16.1'
```

Comment:
There is also https://github.com/electron/electron/releases/tag/v27.0.0 which includes Node 18.17.1.

Note that I'm thinking of introducing some cleanup code for old versions so I might want to introduce all those changes together. Though I'm not sure how I'll do it yet.

Reply:
Would need to be somewhat lazy as upon update older releases may still be running and therefore locked on Windows, so removing immediately might fail.

Reply:
Package updates typically happen on startup and this cleanup would happen before servers are started so it wouldn't be an issue.

It could be an issue when triggering package update manually but then that already can be problematic so I wouldn't worry too much about that. It should correct itself on next start.